src/common/xBuild.ts

```diff
@@ -6,7 +6,7 @@ type XmlElement = {
   [tag: string]: any;
 };
 
-const nsMap = mapKeys(namespaces, (v, k) => {
+const nsMap = () => mapKeys(namespaces, (v, k) => {
```

Comment:
@sedenardi, to verify that this change is necessary, first request
```
curl -i -L -X REPORT \
   -H "Content-Type:application/xml" \
   -d \
'<?xml version="1.0" encoding="UTF-8"?>
<L:calendar-query xmlns:L="urn:ietf:params:xml:ns:caldav">
  <D:prop xmlns:D="DAV:">
    <D:getcontenttype/>
    <D:getetag/>
    <L:calendar-data/>
  </D:prop>
  <L:filter>
    <L:comp-filter name="VCALENDAR">
      <L:comp-filter name="VEVENT">
        <L:time-range start="20181025T000000Z"/>
      </L:comp-filter>
    </L:comp-filter>
  </L:filter>
</L:calendar-query>' \
 'http://localhost:3001/caldav/cal/user@ex.co/exampleCal1'
```
and then repeat the request with
```
<L:time-range start="20281025T000000Z"/>
```
The first request should return three events, the second none. But prior to @aharter's change, the second request gave the same response as the first, because `nsMap` had the same value on both occasions.

Reply:
Ahh. So the issue is before the change, the response for both of your requests returns 0 events:

```xml
<?xml version="1.0"?>
<D:multistatus xmlns:D="DAV:" xmlns:CAL="urn:ietf:params:xml:ns:caldav" xmlns:CS="http://calendarserver.org/ns/" xmlns:ICAL="http://apple.com/ns/ical/"/>
```

Whereas with this PR, the first request correctly returns 3 events, and the second request correctly returns 0 events.